tripal_analysis_expression/includes/HeatMapGenerator.inc

```diff
@@ -116,7 +116,7 @@ class HeatMapGenerator{
             INNER JOIN {biomaterial} B ON B.biomaterial_id = AB.biomaterial_id
             INNER JOIN {analysis} AN ON AN.analysis_id = Q.analysis_id
             LEFT JOIN {contact} C ON B.biosourceprovider_id = C.contact_id
-            WHERE F.uniquename IN (:uniquenames)";
+            WHERE F.name IN (:uniquenames)";
```

Comment:
Selecting on the 'name' vs the 'uniquename'  is an important distinction.  If we just change it like this it may break someone else's site.  I can see that folks may want both.  Maybe this function should be changed to have an input argument that indicates of searching should occur using the `name` or the `uniquename` field and it can default to the `uniquename` field to ensure backwards compatibility.